src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -0,0 +1,12 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class ColorSupplier {
+    private final Random random = new Random();
+
+    public String getRandomColor() {
+        int randomColor = random.nextInt(Colors.values().length);
+        return Colors.values()[randomColor].name();
```

Comment:
Good usage of the 'name()' method for getting the String representation of enum constants, which adheres to the best practices outlined in the checklist.